assembly/src/ast/imports.rs

```diff
@@ -88,8 +96,8 @@ impl ModuleImports {
     }
 
     /// Look up the path of the imported module with the given name.
-    pub fn get_module_path(&self, module_name: &str) -> Option<&LibraryPath> {
-        self.imports.get(&module_name.to_string())
+    pub fn get_module_path(&self, module_name: &str) -> Option<LibraryPath> {
+        self.imports.get(&module_name.to_string()).cloned()
     }
```

Comment:
Why do we need to change this?

Reply:
I need to clone the module path to be able to insert in to the `reexported_modules`, otherwise I will try to mutate `import_info`, having immutable reference to it. I use it in the context.rs file